http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/BaseHttpAdvancedReactiveIT.java

```diff
@@ -182,6 +182,43 @@ public void multipartFormDataReader() {
                 .body(TEXT, is(TEXT));
     }
 
+    @Test
+    @DisplayName("RESTEasy Reactive Multipart Max Parameters test")
+    public void multiPartFormDataMaxParametersAllowed() {
+        final int MAX_PARAMETERS_ALLOWED = 999;
+        var request = getApp().given().multiPart(FILE, Paths.get("src", "test", "resources", "file.txt").toFile());
+
+        for (int i = 1; i < MAX_PARAMETERS_ALLOWED; i++) {
```

Comment:
It seems there is a off-by-one error. There would be 998 parameters, from param1 to param998. Either start from 0, or make the comparison inclusive.

Same goes for the loop below.

Reply:
I know, what you're checking, but you're doing it incorrectly (or the description is wrong):
```
        final int MAX_PARAMETERS_ALLOWED = 999;
        int total = 0;
        for (int i = 1; i < MAX_PARAMETERS_ALLOWED; i++) {
            total += 1;
            System.out.print(i + " ");
        }
        System.out.println("total: " + total);
```
output:
```
1 <...> 998
total: 998
```

Reply:
Yeah, I knew it, so there is an extra parameter (content) that I didn't expect here (and thanks for reviewing this) in the multi-part and it's the "src/test/resources/file.txt" so that's 1 parameter more to add and the if I do : 
```
final int MAX_PARAMETERS_ALLOWED = 999;
 for (int i = 0; i < MAX_PARAMETERS_ALLOWED; i++) {
```
The test will fail because count in total 1001 parameter when 1000 is the limit. :)